src/explore-education-statistics-common/src/modules/charts/components/__tests__/ChartRenderer.test.tsx

```diff
@@ -75,4 +79,42 @@ describe('ChartRenderer', () => {
     );
     expect(screen.queryByTestId('footnotes')).not.toBeInTheDocument();
   });
+
+  test('shows an error message with bad chart data ', () => {
+    const invalidTestChartRenderer: ChartRendererProps = {
+      type: 'verticalbar',
+      meta: testFullTableMeta.subjectMeta,
+      data: testFullTableMeta.results,
+      alt: '',
+      height: 100,
+      axes: undefined as unknown as LineChartProps['axes'],
+      legend: undefined as unknown as LegendConfiguration,
+    };
+
+    render(<ChartRenderer id="test-id" {...invalidTestChartRenderer} />);
+    expect(
+      screen.getByText('Unable to render chart, chart incorrectly configured'),
+    ).toBeInTheDocument();
+  });
+
+  test('shows an error message with bad map data ', () => {
```

Comment:
Similar to earlier comment - this error handling isn't very robust and prevents us from tracking these issues in Application Insights.

Would suggest that in these situations, we just allow the `ChartRenderer` to throw an error with an appropriate message. This would then be caught and logged by the nearest error boundary.